vtStorAta/AtaCommandExtensionscpp.cpp

```diff
@@ -206,5 +206,32 @@ namespace vtStor
 
             return(Drive->IssueCommand(CommandType, commandDescriptor, Data));
         }
+
+        eErrorCode cAtaCommandExtensions::IssueCommand_ATATrim(std::shared_ptr<IDrive> Drive, U32 CommandType, std::shared_ptr<IBuffer> Data, U16 Count, std::vector<sLBARangeEntry> LBARangeEntryList)
```

Comment:
Do we need the parameter Count? Can you calculate it based on the LBARangeEntryList?
About LBARangeEntryList, how about if the name is LBARangeEntries?